src/model/test/People_GTest.cpp

```diff
@@ -40,6 +40,18 @@ TEST_F(ModelFixture, People_DefaultConstructor) {
   EXPECT_EQ(0, definition.numberofPeople().get());
   EXPECT_FALSE(definition.peopleperSpaceFloorArea());
   EXPECT_FALSE(definition.spaceFloorAreaperPerson());
+
+  EXPECT_TRUE(definition.isMeanRadiantTemperatureCalculationTypeDefaulted());
+  EXPECT_NE("ZoneAveraged", definition.meanRadiantTemperatureCalculationType());
+  EXPECT_EQ("EnclosureAveraged", definition.meanRadiantTemperatureCalculationType());
+  EXPECT_TRUE(definition.setMeanRadiantTemperatureCalculationType("ZoneAveraged"));
+  EXPECT_EQ("ZoneAveraged", definition.meanRadiantTemperatureCalculationType());
```

Comment:
Oh, I think I see what you tried to do in the FT and IDD by keeping "ZoneAveraged". You were trying to allow backward compat with `definition.setMeanRadiantTemperatureCalculationType("ZoneAveraged")`

I was expecting `setMeanRadiantTemperatureCalculationType` to automatically replace ZoneAveraged with EnclosureAverage instead. That leaves the getter returning a new value, and it would break code that does something like `if def.meanRadiantTemperatureCalculationType.downcase == 'zoneaverage'` but I don't think we really care about this. I saw no OS gems that actually used that method.